aip-console-tools-cli/src/test/java/com/castsoftware/aip/console/tools/CreateApplicationCommandTest.java

```diff
@@ -0,0 +1,105 @@
+package com.castsoftware.aip.console.tools;
+
+import com.castsoftware.aip.console.tools.commands.CreateApplicationCommand;
+import com.castsoftware.aip.console.tools.factories.SpringAwareCommandFactory;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import picocli.CommandLine;
+import picocli.spring.PicocliSpringFactory;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+//@ExtendWith(MockitoExtension.class)
+public class CreateApplicationCommandTest {
+    private static final String TEST_CREATRE_APP = "To_Create_App-name";
+    private static final String TEST_API_KEY = "API-Key";
+
+    private AnnotationConfigApplicationContext context;
+    private CommandLine createAppCmdLine;
+    private CreateApplicationCommand createAppCmd;
+
+    @Before
+    public void startup() {
+        load(AipIntegrationCliMain.class);
+        CommandLine.IFactory factory = this.context.getBean(SpringAwareCommandFactory.class);
+        createAppCmd = this.context.getBean(CreateApplicationCommand.class);
+        createAppCmdLine = new CommandLine(createAppCmd, factory);
+    }
+
+    @After
+    public void tearDown() {
+        if (this.context != null) {
+            this.context.close();
+        }
+    }
+
+    @Test
+    public void testSpringAwareCommandFactory() {
```

Comment:
This test is unnecessary.
Spring knows how to instantiate classes, no need to check this manually, this is not our responsibility.